openapi/golem-service.yaml

```diff
@@ -3388,6 +3388,170 @@ paths:
             application/json; charset=utf-8:
               schema:
                 $ref: '#/components/schemas/ErrorBody'
+  /v1/library-plugins:
```

Comment:
An alternative api would be to have a merged ComponentTransformer plugin type and have a nested type to encode the direction of the composition.

I feel this way will be easier to explain in docs / more intuitive for users though